.github/workflows/build_and_test_component.yml

```diff
@@ -0,0 +1,56 @@
+run-name: Build and Test Component - Build ${{ inputs.component }}
+name:  Build and Test Component
+
+on:
+  workflow_dispatch:
+    inputs:
+      source_reference:
+        description: |
+          Branch from wazuh/wazuh-agent repository to use.
+        required: true
+      component:
+        description: 'Default agent'
+        required: true
+        type: choice
+          - agent
+        default: agent
+
+  workflow_call:
+    inputs:
+      source_reference:
+        type: string
+        required: true
+      components:
```

Comment:
This input variable has an `s` in the end, shouldn't it be the same as the one used in the `workflow_dispatch` part?